### ISSUE TITLE
Lightning Input Type = datetime does not render with LDS locally, but it does on the Server

### ISSUE THREAD

**Issue by AllanOricil** [forcedotcom/lwc-dev-server-feedback#6]
**Describe the bug**
Lightning Input Type = datetime does not render with LDS locally, but it does on the Server.

On my computer it renders like below:
![image](https://user-images.githubusercontent.com/55927613/66254906-ea9ea980-e774-11e9-8a45-ef0cb461f053.png)

On the server it renders like below:
![image](https://user-images.githubusercontent.com/55927613/66254912-f7230200-e774-11e9-9a50-90bfbcafbfd0.png)

**To Reproduce**
use the following tag in your lwc and run it locally:
```html
<lightning-input type="datetime" label="Due To" value={_date}></lightning-input>
```

**Expected behavior**

**Desktop (please complete the following information):**
 - OS: windows 10
 - Browser Chrome
 - Version Last Version

**Comment by lcampos** [comment]
This was fixed as part of release 2.1.0 and above. Thanks for opening this issue.

**Comment by AllanOricil** [comment]
Great. Thank you.
